packages/x-components/src/x-modules/semantic-queries/components/semantic-queries.vue

```diff
@@ -1,14 +1,34 @@
 <template>
-  <NoElement v-if="$x.semanticQueries.length">
-    <slot name="default" :queries="queries">
+  <NoElement v-if="semanticQueries.length">
```

Comment:
I used the `NoElement` there because in Vue you can't use a slot as a component root.
About the `v-if`, is there any case where we would need to render the component if there are no queries in the state? 🤔